### PR TITLE
Update tmr-meeting-recording-change.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6870
Added note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tmr-meeting-recording-change.md [hunk 1]
```diff
@@ -34,6 +34,11 @@ appliesto:
 
 Microsoft Teams has a new method for saving meeting recordings. As the first phase of a transition from classic Microsoft Stream to the [new Stream](/stream/streamnew/new-stream), this method stores recordings on Microsoft OneDrive for Business and SharePoint in Microsoft 365 and offers many benefits.
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:37]:
```suggestion
> [!NOTE]
```

File: Teams/tmr-meeting-recording-change.md [hunk 2]
```diff
@@ -34,6 +34,11 @@ appliesto:
...
+> If a Teams meeting recording fails to successfully upload to OneDrive/SharePoint, the recording will instead be temporarily saved to Azure Media Services (AMS). Once stored in   AMS, no retry attempts are made to automatically upload the recording to OneDrive/SharePoint or Stream.
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:38]:
```suggestion
> If a Teams meeting recording fails to successfully upload to OneDrive/SharePoint, the recording will instead be temporarily saved to Azure Media Services (AMS). Once stored in AMS, no retry attempts are made to automatically upload the recording to OneDrive/SharePoint or Stream.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**